api/v1alpha1/externaldns_types.go

```diff
@@ -457,7 +457,10 @@ type ExternalDNSSourceUnion struct {
 	// +optional
 	CRD *ExternalDNSCRDSourceOptions `json:"crd,omitempty"`
 
-	// OpenShiftRoute source currently has no unique configuration options.
+	// OpenShiftRoute source configuration options for specifying ingress controller names.
+	// +kubebuilder:validation:Optional
+	// +optional
+	OpenShiftRoute *ExternalDNSServiceOpenShiftRouteOptions `json:"openshift_route,omitempty"`
```

Comment:
```suggestion
	OpenShiftRoute *ExternalDNSOpenShiftRouteOptions `json:"openshiftRoute,omitempty"`
```
The word _"Service"_ in the name of the type is not required. The options are for a type of `ExternalDNS`. The name of the JSON field should use camel-case because that's the Kubernetes/Openshift convention.